packages/dev/test/NetworkSimultaion.test.ts

```diff
@@ -161,7 +154,48 @@ contract('Network Simulation for Relay Server', function (accounts) {
       })
     })
 
-    async function sendMultipleRelayedTransactions (): Promise<void> {
+    describe('boosting & rebroadcasting all transactions', function () {
+      let id: string
+      before(async () => {
+        id = (await snapshot()).result
+      })
+      after(async () => {
+        await revert(id)
+      })
+      it('should boost underpriced transactions and only rebroadcast fairly priced transactions', assertBoostAndRebroadcast)
+      it('should not boost any transactions if latest on-chain nonce is lower than oldest pending tx nonce', async function () {
+        const storedTxs = await env.relayServer.txStoreManager.getAll()
+        const latestNonce = await env.web3.eth.getTransactionCount(storedTxs[0].from)
+        assert.equal(storedTxs[0].nonce, latestNonce)
+        await env.relayServer.txStoreManager.removeTxsUntilNonce(storedTxs[0].from, latestNonce)
+        await evmMineMany(pendingTransactionTimeoutBlocks)
+        try {
+          await env.relayServer._boostStuckPendingTransactions(await env.web3.eth.getBlockNumber())
+          assert.fail()
+        } catch (e) {
+          assert.equal(e.message, `Boosting: missing nonce ${latestNonce}. Lowest stored tx nonce: ${storedTxs[1].nonce}`)
+        }
+      })
+      it('should not boost any transactions if config.pendingTransactionTimeoutBlocks did not pass yet', async function () {
+        await env.relayServer.txStoreManager.clearAll()
+        await env.relayServer.transactionManager._initNonces()
+        await sendMultipleRelayedTransactions(stuckTransactionsCount)
+
+        const storedTxs = await env.relayServer.txStoreManager.getAll()
+        const latestNonce = await env.web3.eth.getTransactionCount(storedTxs[0].from)
+        assert.equal(storedTxs[0].nonce, latestNonce)
+        const spy = sinon.spy(env.relayServer.logger, 'debug')
+        const message = `${storedTxs[0].from} : awaiting transaction with ID: ${storedTxs[0].txId} to be mined. creationBlockNumber: ${storedTxs[0].creationBlockNumber} nonce: ${storedTxs[0].nonce}`
+        await env.relayServer._boostStuckPendingTransactions(await env.web3.eth.getBlockNumber())
+        spy.calledWith(message)
```

Comment:
This line does not look like a valid assertion. I believe `calledWith` returns a value, and it is your job to write an assertion (isTrue, isFalse, throws etc.)